File: 250323/10개 입력받기/receive-10-inputs.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int[] arr = new int[10];
        int count = 0;
        for(int i = 0; i < 10; i++) {
            arr[i] = sc.nextInt();
            if(arr[i] == 0) {
                break;
            }
            count++;
        }
        int sum = 0;
        double avg = 0;
        for(int i = 0; i < count; i++) {
            sum += arr[i];
        }
        avg = (double)sum / count;
        System.out.printf("%d %.1f", sum, avg);
        sc.close();
    }
}

File: 250324/10개 입력받기/receive-10-inputs.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int sum = 0, count = 0;
    double avg = 0;
    int arr[10] = {0,};
    for(int i = 0; i < 10; i++) {
        cin >> arr[i];
        if(arr[i] == 0) {
            break;
        }
        sum += arr[i];
        count++;
    }
    avg = (double)sum / count;
    cout << fixed;
    cout.precision(1);
    cout << sum << " " << avg;
    return 0;
}